streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -340,6 +319,27 @@ public TaskAssignor getTaskAssignor() {
         }
     }
 
+    public AssignmentListener assignmentListener() {
```

Comment:
This is the only non-cosmetic change in this class, along with the interface added below